public/support/mdb_operator_diagnostic_data.sh

```diff
@@ -2,6 +2,45 @@
 
 set -Eeou pipefail
 
+# Check if oc and kubectl exist
+command -v oc >/dev/null 2>&1
+ocExists=$?
+
+command -v kubectl >/dev/null 2>&1
+kcExists=$?
+
+# Let the user override via environment variable
+dToolUser=${DTOOL:-""}
+dToolUserArgs=${DTOOLARGS:-""}
```

Comment:
what exactly is the motivation to introduce this? do we want users to be able to configure the kubeconfig?